pp_calibrate.py

```diff
@@ -478,14 +478,26 @@ def calibrate(filenames, minstars, manfilter, manualcatalog,
     # read in ldac data into catalogs
     catalogs, filternames = [], {}
     for filename in filenames:
-        hdulist = fits.open(filename, ignore_missing_end=True)
+        hdulist = fits.open(filename, mode='update', ignore_missing_end=True)
+        
+        # add flag keyword to header and set it to STARTED
+        header = hdulist[0].header
+        try:
+            header.set('PP_CALIB', 'STARTED', 'PP: pp_calibrate status flag',
```

Comment:
thinking about it, shouldn't we set the keyword to `FAILED` by default? If the function breaks, it will stick with `FAILED`, or else it will switch to `SUCCESS`. Can you change that in this function and the others?